api/init/03-poisInit.sql

```diff
@@ -1,9 +1,10 @@
 DROP TABLE IF EXISTS POIs CASCADE;
 
-CREATE TYPE poi_type_enum AS ENUM ('Trailhead', 'Trail Location');
+CREATE TYPE poi_type_enum AS ENUM ('Trailhead', 'Peak', 'Scenic', 'Lodge', 'Leanto');
 
 CREATE TABLE POIs (
     id SERIAL PRIMARY KEY,
     name TEXT,
-    type poi_type_enum
+    type poi_type_enum,
+    wilderness_area TEXT
```

Comment:
TBH it would probably be more future-proof to have the wilderness area be an ID key into a new WildernessAreas table which is just a bunch of rows with IDs and Names for the different wilderness areas.